rag_experiment_accelerator/config/config.py

```diff
@@ -32,6 +32,9 @@ class Config:
     Attributes:
         CHUNK_SIZES (list[int]): A list of integers representing the chunk sizes for chunking documents.
         OVERLAP_SIZES (list[int]): A list of integers representing the overlap sizes for chunking documents.
+        GENERATE_TITLE (bool): Whether or not to generate title for chunk content. Default is False.
```

Comment:
why have you moved these comments?
(just curious)

Reply:
because they are related to chunking strategies, I thought it would be good that they would be together :)